src/internal/packager2/inspect.go

```diff
@@ -305,6 +418,50 @@ func getTemplatedManifests(ctx context.Context, manifest v1alpha1.ZarfManifest,
 	return resources, nil
 }
 
+// loadInspectPackageLayout replicates the Load process while adding inspect targets not utilized in the primary LoadPackage function.
+func loadInspectPackageLayout(ctx context.Context, source, architecture string, inspectTarget InspectTarget, cluster *cluster.Cluster, filter filters.ComponentFilterStrategy, publicKeyPath string, skipSignatureValidation bool) (*layout.PackageLayout, error) {
```

Comment:
Thoughts on `LoadPackage` accepting an `InspectTarget` as an option? I believe this function is the same as `LoadPackage` besides that

Reply:
That was my original take - keeping `LoadPackage` as the sole-source for package load functionality. Feedback provided was to break `LoadPackage` into multiple private functions and then calling them from different public functions. 

Given the validation now is encapsulated by custom types and filter processes accordingly I am going to play with adding it back.

Reply:
Updated